app/models/manageiq/providers/ansible_tower/shared/automation_manager.rb

```diff
@@ -19,6 +19,9 @@ def connection_source(options = {})
     end
   end
 
+  alias_attribute :playbooks, :configuration_script_payloads
```

Comment:
Is this the way we alias associations? I guess this has problems when reflecting on them.
I would have guessed you add a new association.

@kbrock do you know?